README.md

```diff
@@ -78,19 +88,21 @@ struct RouteLabels: MetricLabels {
    var route: String = "*"
 }
 
-let counter = Prometheus.shared.createCounter(forType: Int.self, named: "my_counter", helpText: "Just a counter", withLabelType: RouteLabels.self)
+let prom = PrometheusClient()
+
+let counter = prom.createCounter(forType: Int.self, named: "my_counter", helpText: "Just a counter", withLabelType: RouteLabels.self)
 
 counter.inc(12, .init(route: "/"))
 ```
 
 # Exporting
 
-To keep SwiftPrometheus as clean and leight weitght as possible, there is no way of exporting metrics to Prometheus. All you can do is get a formatted string that Prometheus can use, so you can integrate it in your own Serverside Swift application
+To keep SwiftPrometheus as clean and lightweight as possible, there is no way of exporting metrics directly to Prometheus. Instead, retrieve a formatted string that Prometheus can use, so you can integrate it in your own Serverside Swift application
 
 This could look something like this:
 ```swift
 router.get("/metrics") { request -> String in
-    return Prometheus.shared.getMetrics()
+    return prom.getMetrics()
```

Comment:
This won't work with the new async API.
The request would also have to return `Future<String>` but I can not put that in the suggestion :)
```suggestion
    let promise = req.eventLoop.newPromise(String.self)
    prom.getMetrics {
        promise.succeed(result: $0)
    }
    return promise.futureResult
```